RobotsWantedLeague/Services/RobotsService.cs

```diff
@@ -1,59 +1,50 @@
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+
 namespace RobotsWantedLeague.Services;
 
 using RobotsWantedLeague.Models;
 
 public class RobotsService: IRobotsService
 {
-    private readonly List<Robot> robots;
-    private int idGenerator = 0;
-    public List<Robot> Robots { get => robots; }
+    private readonly IMongoCollection<Robot> robotsCollection;
```

Comment:
So instead of a `List<robot>` we now depend on a `IMongoCollection<Robot>`.  The api of `IMongoCollection<Robot>` is quite nice.